### PR TITLE
CPLAT-4956 CPLAT-4955 Release over_react 2.4.0+dart2

### PR DESCRIPTION
**Pull request by rmconsole-readonly-wk**
Pull Requests included in release:
* Patch changes:
	* [CPLAT-3300: CP Code Review and PR Merging Team agreement update](https://github.com/Workiva/over_react/pull/277)
	* [CPLAT-5152: Optimize builder's use of asset reads to avoid unnecessary rebuilds](https://github.com/Workiva/over_react/pull/280)

Requested by: @evanweible-wf

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/over_react/compare/2.3.1+dart2...Workiva:release_over_react_2.4.0+dart2
Diff Between Last Tag and New Tag: https://github.com/Workiva/over_react/compare/2.3.1+dart2...2.4.0+dart2

The logs for the request that created this PR can be found [here](https://w-rmconsole.appspot.com/release/release_event/6449952306233344/logs/)

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+## 2.4.0
+
+> Complete `2.4.0` Changsets:
+>
+> - [Dart 2](https://github.com/Workiva/over_react/compare/2.3.1+dart2...2.4.0+dart2)
+> - Dart 1 (no changes)
+
+* [#280] Optimize the over_react builder to avoid unnecessary asset reads.
+  Informal profiling in a large codebase that consumes over_react has shown in
+  the best case a speed up on rebuilds from ~1 minute to ~2 seconds.
```
Comment (corwinsheahan-wf) [inline CHANGELOG.md]: #nit Might be worth noting that no trials showed a decrease in performance.

Comment (evanweible-wf) [reply CHANGELOG.md]: Addressed in c5c2856

**Comment by evanweible-wf** [comment]
QA +1

- [x] Dev +1's only needed for Release ticket

@Workiva/release-management-pp

**Pull request merged**